.pug-lintrc.json

```diff
@@ -1,19 +1,20 @@
-{  
+{
     "disallowAttributeInterpolation": true,
-    "disallowDuplicateAttributes": true, 
+    "disallowDuplicateAttributes": true,
     "disallowMultipleLineBreaks": true,
-    "disallowStringConcatenation": true, 
+    "disallowStringConcatenation": true,
     "disallowStringInterpolation": true,
-    "requireClassLiteralsBeforeAttributes": true, 
-    "requireClassLiteralsBeforeIdLiterals": true, 
+    "requireClassLiteralsBeforeAttributes": true,
+    "requireClassLiteralsBeforeIdLiterals": true,
     "requireIdLiteralsBeforeAttributes": true,
     "requireSpaceAfterCodeOperator": true,
     "requireStrictEqualityOperators": true,
-    "validateAttributeSeparator": { 
-        "separator": " ", 
+    "validateAttributeQuoteMarks": "'",
+    "validateAttributeSeparator": {
+        "separator": " ",
         "multiLineSeparator": "\n  "
-    }, 
+    },
     "validateDivTags": true,
-    "validateIndentation": 2, 
+    "validateIndentation": 2,
     "validateSelfClosingTags": true
-}
\ No newline at end of file
+}
```

Comment:
@taoeffect and @pieer I realized we don't run puglint while running the project as we do for style and eslint. Is there any reason for that?

Reply:
I can't think of one. Feel free to add it to the list of linters that gets run on file change

Reply:
Which list is that exactly? I saw this but pug is there...

```
grunt.task.run(['exec:eslint', 'exec:puglint', 'exec:stylelint', 'copy', 'sass', rollup])
```

What is missing here? 🤔

Reply:
@sandrina-p I've figured out why it's not working, it's for two reasons:

1. Normally, the way this would work is that `grunt-contrib-watch` would fire off a task (look at the `watch` configuration), however, because of the way rollup works and how we've created a custom task for it that does its own watching of the files, this no longer happens
2. The custom `rollup` task is what launches the rollup watcher via `rollup.watch(watchOpts)`, and `plugins` that we give to rollup do not include a plugin to lint the pug source (neither `VuePlugin` nor `eslint` plugins run on the `<template lang='pug'>` stuff)

So there are two potential fixes, an easy one and a hard one:

1. Easy fix: more of a hack than a fix, to the `watch` config section on `rollup`, create a `tasks` key like this:

    ```
    tasks: ['exec:puglint'] // hack, correct way to do this would be via rollup.watch
    ```

2. Hard fix: either figure out what options you need to pass to either `rollup-plugin-eslint` or `rollup-plugin-vue` to make them also run puglint, or create a custom rollup plugin function (like the `flow` plugin function at the bottom of `.Gruntfile.babel.js`) that manually runs puglint (maybe research how `eslint` plugin works before doing this, in order to get an idea of how the plugin function should be defined)

You can maybe do the easy fix for this PR and create an issue to do the hard version later, or whatever you decide, this is just a suggestion.

Reply:
The quick fix is done. I tried the hard fix but there isn't a rollup plugin for pug and I didn't find any pug documentation to use it programmatically  :/